projects/ardin_mb/CMakeLists.txt

```diff
@@ -4,13 +4,19 @@ include(../../cmake/bob_robotics.cmake)
 if(NO_GENN)
     add_definitions(-DNO_GENN)
     BoB_project(EXECUTABLE ardin_mb
-                SOURCES ardin_mb.cc state_handler.cc opencv_texture.cc vector_field.cc visual_navigation_ui.cc
+                SOURCES ardin_mb.cc state_handler.cc opencv_texture.cc
+                        vector_field.cc visual_navigation_ui.cc
                 BOB_MODULES common antworld navigation
                 THIRD_PARTY imgui)
 else()
     BoB_project(EXECUTABLE ardin_mb
-                SOURCES ardin_mb.cc state_handler.cc mb_memory.cc opencv_texture.cc mb_memory_ardin.cc vector_field.cc visual_navigation_ui.cc
+                SOURCES ardin_mb.cc state_handler.cc mb_memory.cc
+                        opencv_texture.cc mb_memory_ardin.cc vector_field.cc
+                        visual_navigation_ui.cc
                 BOB_MODULES common antworld navigation
-                THIRD_PARTY imgui
-                INCLUDE_GENN_USERPROJECTS)
+                GENN_MODEL model_ardin.cc
```

Comment:
These weren't included for a reason here - this example uses runtime dynamic linking so it has a plugin style architecture where it loads dynamic libraries previously built by GeNN. That clearly doesn't play nice with CI so, personally, I'd just skip this one. The stone CX model is a much simpler static linking against a GeNN model where this automagical stuff should work. In GeNN I just add files called "skip" to directories with tests I don't wanna run and detect them in [bash](https://github.com/genn-team/genn/blob/master/tests/run_tests.sh#L61-L62)

Reply:
Ahh I see... Ok I won't test this project then